app/controllers/claims/support/school_suggestions_controller.rb

```diff
@@ -0,0 +1,11 @@
+class Claims::Support::SchoolSuggestionsController < ApplicationController
```

Comment:
Given how this is identical to the `Placements::Support::SchoolSuggestionsController`, is it worth having a shared `SchoolSuggestionsController`?

Reply:
Yeah, I think so